### DISCUSSION TITLE
Keep HTML self closing slash on next built

### DISCUSSION THREAD

**Discussion by brosbunbo** [vercel/next.js#27959] [Help]
**Version**: 10

**Current**:
`<link ... />` is converted to `<link ... >` after running `next build`

**Expect**:
`<link ... />` still has self closing slash

How could I achieve this?
Thank and best Regards,

**Comment by delbaoliveira** [comment]
Hi @brosbunbo 👋🏼 

I was not able to reproduce this. Could you expand on where you're seeing this behavior? What are you using to determine there is no self-closing slash? For example, Chrome developer tools doesn't render a self-closing tag even though the view source does. 

It's worth noting that `<link />` is a void element in HTML5 meaning the [self-closing slash is optional](https://dev.w3.org/html5/html-author/#optional-tags) (see 3.2.2.7 Element Type Comparison). Same with `<image>` and `<image />` for example. Is there are a reason for keeping the trailing slash?
